src/config/dynamic-theme-fixes.config

```diff
@@ -8746,6 +8746,13 @@ img[alt="logo"]
 
 ================================
 
+elektronik-kompendium.de
+
+INVERT
+img[alt="Formel"]
```

Comment:
I checked some problematic graphs and formulas and I did not found that alt="Formel" in them so I suggest to change it into:
`img[src*="formel/"]`
as all problematic pictures are stored in this folder on the server.

Reply:
You are are right it should have been `img[src*="formel/"]`.
I also noticed that circuit diagrams weren't looking correctly.

Now I just have to figure out how to add the changes to the pull request